src/operator/tensor/elemwise_unary_op_basic.cc

```diff
@@ -70,7 +70,7 @@ static bool IdentityAttrLikeRhsStorageType(const nnvm::NodeAttrs& attrs,
 }
 
 // relu
-MXNET_OPERATOR_REGISTER_UNARY(relu)
+MXNET_OPERATOR_REGISTER_UNARY_WITH_RSP_CSR(relu, cpu, mshadow_op::relu)
```

Comment:
We probably want to move MXNET_ADD_SPARSE_OP_ALIAS inside MXNET_OPERATOR_REGISTER_UNARY_WITH_RSP_CSR. Do you mind changing that?

Reply:
Actually none of MXNET_OPERATOR_REGISTER_UNARY_WITH_RSP_CSR/MXNET_OPERATOR_REGISTER_UNARY_WITH_RSP/MXNET_OPERATOR_REGISTER_UNARY_WITH_SPARSE_DR has MXNET_ADD_SPARSE_OP_ALIAS in them, do you want to add that to all of them?

Reply:
Add for `MXNET_OPERATOR_REGISTER_UNARY_WITH_RSP_CSR` and  `MXNET_OPERATOR_REGISTER_UNARY_WITH_RSP`. Lets skip the ones with SPARSE_DR

Reply:
Done